ocaml/xapi-cli-server/records.ml

```diff
@@ -1627,6 +1627,98 @@ let subject_record rpc session_id subject =
       ]
   }
 
+let session_record rpc session_id session =
```

Comment:
I think we don't need this at all. It doesn't look like it is even used? We shouldn't have `xe session-list` or similar.

Reply:
This is needed for autogeneration of the CLI commands *-list *-param-get